website/pages/docs/drivers/docker.mdx

```diff
@@ -743,7 +743,7 @@ plugin "docker" {
     `credHelpers` or `credsStore`.
   - `helper`<a id="plugin_auth_helper"></a> - Allows an operator to specify a
     [credsStore](https://docs.docker.com/engine/reference/commandline/login/#credential-helper-protocol)
-    -like script on \$PATH to lookup authentication information from external
+    like script on `\$PATH` to lookup authentication information from external
```

Comment:
```suggestion
    like script on `$PATH` to lookup authentication information from external
```

Reply:
the slash doesn't hurt anything, it's just docs. but it looks funny, i feel like we should remove it while we're here.